lte/gateway/c/core/oai/include/mme_app_statistics.h

```diff
@@ -21,11 +21,11 @@
 /*********************************** Utility Functions to update
  * Statistics**************************************/
 void update_mme_app_stats_connected_ue_add(void);
-void update_mme_app_stats_connected_ue_sub(void);
 void update_mme_app_stats_s1u_bearer_add(void);
 #ifdef __cplusplus
 extern "C" {
 #endif
```

Comment:
Migration of files under include folder will be taken up as a separate activity as these files will be included in many .c/.cpp files

Reply:
Separate PR is fine, but it will good if we could do before porting hash table to protobuf map. Because on inclusion of c++ provided features we get lot of errors as some piece of code is in 'C'